tests/base.py

```diff
@@ -40,7 +40,8 @@ def get_properties(self, original: bool = True):
         return_value = {
             'start_date': '2017-07-01T00:00:00Z',
             'shop': 'stitchdatawearhouse',
-            'date_window_size': 30
+            'date_window_size': 30,
+            'results_per_page': '50'
```

Comment:
It feels weird that results_per_page is a string but date_window_size is an integer. Not sure if that really matters outside of tap-tester though.